### ISSUE TITLE
Setup is not compatible with latest Markdown release

### ISSUE THREAD

**Issue by horazont** [xsf/xmpp.org#468]
See #467 for the symptoms. The symptoms started with Markdown 3.0. Downgraded the build to the latest known-to-work version (2.6.11, see dockerfile).

We need to upgrade our content to make it compatible with current versions.

**Comment by wurstsalat3000** [comment]
This works for me using Pelican 4.2 and Markdown 3.1.1.

```
DEBUG: Successfuly imported extension module "markdown.extensions.codehilite".
DEBUG: Successfully loaded extension "markdown.extensions.codehilite.CodeHiliteExtension".
DEBUG: Successfuly imported extension module "markdown.extensions.extra".
DEBUG: Successfully loaded extension "markdown.extensions.fenced_code.FencedCodeExtension".
DEBUG: Successfully loaded extension "markdown.extensions.footnotes.FootnoteExtension".
DEBUG: Successfully loaded extension "markdown.extensions.attr_list.AttrListExtension".
DEBUG: Successfully loaded extension "markdown.extensions.def_list.DefListExtension".
DEBUG: Successfully loaded extension "markdown.extensions.tables.TableExtension".
DEBUG: Successfully loaded extension "markdown.extensions.abbr.AbbrExtension".
DEBUG: Successfully loaded extension "markdown.extensions.extra.ExtraExtension".
DEBUG: Successfuly imported extension module "markdown.extensions.meta".
DEBUG: Successfully loaded extension "markdown.extensions.meta.MetaExtension".
```

I haven’t found missing pages so far, and the log does not contain any 'WARNING: Could not process' lines.

**Comment by wurstsalat3000** [comment]
Note that I also fixed #506 in my local install. So if this actually works correctly, the version pin to Pelican 3.3 / Markdown 2.6.11 might not be necessary anymore. Someone™ would need to verify this.
